Demo/HelloWorld/Client/src/Program.cs

```diff
@@ -0,0 +1,212 @@
+                                                                 null,
+                                                                 // Generate the given number of task requests
+                                                                 Enumerable.Range(0,
+                                                                                  nTasks)
+                                                                           .Select(_ =>
+                                                                                   {
+                                                                                     // Generate result Ids for the submission of the current
+                                                                                     var taskResultIfs = Enumerable.Range(0,
+                                                                                                                          nResultsPerTask)
+                                                                                                                   .Select(i => Guid.NewGuid() + "_" + i)
+                                                                                                                   .ToList();
+
+                                                                                     // Add the result Ids
+                                                                                     resultIds.AddRange(taskResultIfs);
+
+                                                                                     // Task request with payload for the task
+                                                                                     // Also contains the list of results that will be created by the task
+                                                                                     return new TaskRequest
+                                                                                            {
+                                                                                              ExpectedOutputKeys =
+                                                                                              {
+                                                                                                taskResultIfs,
+                                                                                              },
+                                                                                              // Avoid unnecessary copy but data could be modified during sending if the
+                                                                                              // reference to the object is still available
+                                                                                              // This is not the case here
+                                                                                              Payload = UnsafeByteOperations.UnsafeWrap(Encoding.ASCII.GetBytes(input)),
```

Comment:
Not directly related to PR, but I think it could be nice hide this in TaskRequest constructor or something.

Reply:
That would be great. /s ==> https://github.com/aneoconsulting/ArmoniK.Api/pull/39